scripts/live_test/generate_index.py

```diff
@@ -49,6 +49,13 @@ def generate(container, container_url, testdata, USER_REPO, USER_BRANCH, COMMIT_
     cmd = 'az storage blob upload -f index.html -c {} -n index.html --account-name clitestresultstac'.format(container)
     print('Running: ' + cmd)
     os.system(cmd)
+
+    # Upload to latest container if it is a full live test of official repo dev branch
+    if USER_REPO == 'https://github.com/Azure/azure-cli.git' and USER_BRANCH == 'dev' and USER_TARGET == '' and USER_LIVE == '--live':
+        cmd = 'az storage blob upload -f index.html -c latest -n index.html --account-name clitestresultstac'
```

Comment:
This command will overwrite the existing file in the storage container by default?

Reply:
Yes, it will.